nfcore_custom.config

```diff
@@ -85,10 +85,11 @@ profiles {
     rosalind_uge     { includeConfig "${params.custom_config_base}/conf/rosalind_uge.config" }
     sage             { includeConfig "${params.custom_config_base}/conf/sage.config" }
     sahmri           { includeConfig "${params.custom_config_base}/conf/sahmri.config" }
-    sanger           { includeConfig "${params.custom_config_base}/conf/sanger.config"}    
+    sanger           { includeConfig "${params.custom_config_base}/conf/sanger.config"}
     scw              { includeConfig "${params.custom_config_base}/conf/scw.config"}
     seawulf          { includeConfig "${params.custom_config_base}/conf/seawulf.config"}
     seg_globe        { includeConfig "${params.custom_config_base}/conf/seg_globe.config"}
+    software_license { includeConfig "${params.custom_config_base}/conf/software_license.config"}
```

Comment:
wouldn't it be better to be more explicit by calling it `sentieon_software_license.config` ?

Reply:
the idea would be to be future proof and if later we need a licence for let's say parabricks, dragen, or anything else, we have a config ready for it to use at an nf-core level